project.md

```diff
@@ -0,0 +1,10 @@
+# Project docs
+
+## Release Process
+
+* Increment version in version.go
```

Comment:
Does this happen before or after the release?  See #203 bumping well before the 0.2.0 release, and my [comment][1] in #194 suggesting we either use SemVer's pre-release markers or increase our cadence.

[1]: https://github.com/opencontainers/specs/pull/194#issuecomment-142062583